WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/WalletBalanceTileViewModel.cs

```diff
@@ -7,12 +7,17 @@ namespace WalletWasabi.Fluent.ViewModels.Wallets.Home.Tiles;
 
 public class WalletBalanceTileViewModel : ActivatableViewModel
 {
-	public WalletBalanceTileViewModel(WalletViewModel walletVm)
+	public WalletBalanceTileViewModel(WalletViewModel walletVm, bool showOnlyAvailable = false)
 	{
 		var wallet = walletVm.Wallet;
 
-		var balance = walletVm.UiTriggers.BalanceUpdateTrigger
-			.Select(_ => wallet.Coins.TotalAmount());
+		var trigger = walletVm.UiTriggers.BalanceUpdateTrigger;
+		if (showOnlyAvailable)
+		{
+			trigger = trigger.Merge(walletVm.UiTriggers.WalletCoinsCoinjoinTrigger);
+		}
+
+		var balance = trigger.Select(_ => showOnlyAvailable ? wallet.Coins.Available().TotalAmount() : wallet.Coins.TotalAmount());
```

Comment:
I would correct myself and simplify this code (not tested)
```suggestion
		var balance = showOnlyAvailable
	? walletVm.UiTriggers.BalanceUpdateTrigger.Merge(walletVm.UiTriggers.WalletCoinsCoinjoinTrigger).Select(_ => wallet.Coins.Available().TotalAmount())
	: walletVm.UiTriggers.BalanceUpdateTrigger.Select(_ => wallet.Coins.TotalAmount());
```

Reply:
I will check this out.